tests/unit/sagemaker/workflow/helpers.py

```diff
@@ -38,27 +39,48 @@ def ordered(obj):
         return obj
 
 
-def get_step_args_helper(step_args, step_type):
+def get_step_args_helper(step_args, step_type, use_custom_job_prefix=False):
     execute_job_functions(step_args)
     request_args = step_args.func_args[0].sagemaker_session.context.args
 
     if step_type == "Processing":
-        request_args.pop("ProcessingJobName", None)
+        if not use_custom_job_prefix:
```

Comment:
Use a helper function for this repeated logic? I think it would help with readability, preventing copy/paste errors, and reducing the number of changes needed if future updates are required for this logic